crates/swc_plugin_runner/src/cache.rs

```diff
@@ -20,7 +20,7 @@ use wasmer_cache::{Cache as WasmerCache, FileSystemCache, Hash};
 /// however it is not gauranteed to be compatible across wasmer's
 /// internal changes.
 /// https://github.com/wasmerio/wasmer/issues/2781
-const MODULE_SERIALIZATION_VERSION: &str = "v1";
+const MODULE_SERIALIZATION_VERSION: &str = "v2";
```

Comment:
maybe we should've set this like `v000001` and do the last digit increment? should we do it now?